components/Layouts/Sidebar/Sidebar.tsx

```diff
@@ -34,13 +39,33 @@ const Logo = () => {
 
 export const Sidebar = () => {
   const { data: session, status } = useSession();
-  const { pathname } = useRouter();
+  const router = useRouter();
+  const dispatch = useAppDispatch();
+  const message = 'Continue with your social account';
+  const [loginMessage, setLoginMessage] = useState(message);
+  const [isAuthenticated, setIsAuthenticated] = useState(
+    status === 'authenticated'
+  );
+
+  useEffect(() => {
+    setIsAuthenticated(status === 'authenticated');
+  }, [status]);
```

Comment:
remove useeffect

Reply:
it's not resolved. remove the useeffect as well as the state. `isAuthenticated` is not necessary. you can use status variable. I HOPE YOU UNDERSTAND

Reply:
Please PULL THE CHANGES and then check. It has been resolved in commit 34b7d7eee37e2bfadbef7692e690bd6aca90f061 :)

Reply:
OKAY!